docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -63,11 +67,14 @@ You can also specify:
   language to the GitHub code scanning API. For more information about this use case,
   see `Configuring CodeQL CLI in your CI system <https://docs.github.com/en/code-security/secure-coding/using-codeql-code-scanning-with-your-existing-ci-system/configuring-codeql-cli-in-your-ci-system>`__ in the GitHub documentation.
 
-- ``--sarif-add-query-help``: (supported in version 2.7.1 onwards) adds any custom query help written 
-   in markdown to SARIF files (v2.1.0 or later) generated by the analysis. Query help stored in ``.qhelp`` files must be 
-   converted to ``.md`` before running the analysis. For further information, 
+- ``--sarif-add-query-help``: (supported in version 2.7.1 onwards) adds any custom query help written
+   in markdown to SARIF files (v2.1.0 or later) generated by the analysis. Query help stored in ``.qhelp`` files must be
+   converted to ``.md`` before running the analysis. For further information,
    see ":ref:`Including query help for custom CodeQL queries in SARIF files  <including-query-help-for-custom-codeql-queries-in-sarif-files>`."
 
+- ``--download``: a boolean flag that will allow the CLI to download  any referenced CodeQL packages that are not available locally.
+   If this flag is missing and a referenced CodeQL package is not available locally, the command will fail.
```

Comment:
This suffers from the same problem as lines 70/71 in the original article.

Reply:
Looks like there's an extra space at the start of the line. Fixing.